tests/functional/spec/features/last_viewed_guidance/last_viewed_guidance.spec.js

```diff
@@ -0,0 +1,56 @@
+const JwtHelper = require("../../../jwt_helper");
+
+const AddressConfirmationPage = require('../../../generated_pages/last_viewed_question_guidance/address-confirmation.page.js');
+const HouseholdInterstitialPage = require('../../../generated_pages/last_viewed_question_guidance/household-interstitial.page.js');
+const PrimaryPersonListCollectorPage = require('../../../generated_pages/last_viewed_question_guidance/primary-person-list-collector.page.js');
+
+describe('Last viewed question guidance', function () {
+
+  const last_viewed_question_text = 'This is the last viewed question in this section'
```

Comment:
We should be checking for the panel rather than the text. We've had issues before when we have tried to run functional tests against schemas whilst changing language.